p2p/pex/errors.go

```diff
@@ -63,3 +63,11 @@ type ErrAddrBookInvalidAddr struct {
 func (err ErrAddrBookInvalidAddr) Error() string {
 	return fmt.Sprintf("Cannot add invalid address %v: %v", err.Addr, err.AddrErr)
 }
+
+type ErrAddressBanned struct {
```

Comment:
please add documentation

Reply:
What about for other errors - there isn't any documentation for them either

Reply:
could you please write doc for them too?